src/Volume.jsx

```diff
@@ -38,9 +40,11 @@ export const Volume = ({ id, item, onChange, idx, removeitem, additem, options,
             >
                 <FileAutoComplete id={id + "-host-path"}
                     value={item.hostPath || ''}
-                    onChange={value => {
+                    onChange={(value, error) => {
+                        console.log(`value: ${value}, error: ${error}`);
+                        console.log(validationFailed);
                         utils.validationClear(validationFailed, "hostPath", onValidationChange);
-                        utils.validationDebounce(() => onValidationChange({ ...validationFailed, hostPath: validateVolume(value, "hostPath") }));
+                        utils.validationDebounce(() => onValidationChange({ ...validationFailed, hostPath: validateVolume(error, "hostPath") }));
```

Comment:
This looks strange.. is `onValidationChange()` *only* being called for errors? the code certainly doesn't look like it. Or I don't understand this change just by looking at the patch.

Reply:
No, it is always called onChange?

Reply:
That's what I thought, but now it doesn't even get the value passed any more?

Reply:
So the validation function `validateVolume` usually takes the new value, but the problem is that when an error occurs the value is empty and this is also the initial value.

We could make `validateVolume` check if the file exists, but that is also impossible as a faulty file won't be returned. So instead I pass the `error` to the validation function.

Reply:
Right, I see. validateVolume() does not actually do anything with it, and just passes it back to the caller. So that's fine.